### PR TITLE
Feature/9854 stock photos free other alert

### PR DESCRIPTION
**Pull request by ctarda**
First step in the implementation of #8954 

We have to start somewhere, so here we go. A short PR, that prepares some ground for the rest of the feature.

In this PR:
- Alert presenting options to navigate to Files or the Free Photo Library
- The alert is not constructed in `AztecPostViewController` but in a separate class. I don't think we should add more code to `AztecPostViewController` unless it is strictly necessary.

To test:
1. Start creating a new post
2. Tap the + button in the lower Format Bar.
3. Tap the ellipsis in the Format Bar
4. There should be an alert with three options (Free photo Library, Files, Cancel)
5. It should still be possible to import content from the Files app.
6. Cancel should dismiss the alert
7. Taping "Free Photo Library" should only dismiss the alert but do nothing else.

img_2624
img_2625

### CODE REVIEW

**Comment by iamthomasbishop** [comment]
Thanks for tackling this, @ctarda! One tiny thing:

> There should be an alert with three options (Free photo Library, Files, Cancel)

For consistency, this should probably be:

* Free Photo Library
* Other Apps
* Dismiss

File: WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+        alertController.popoverPresentationController?.sourceView = view
+        alertController.popoverPresentationController?.sourceRect = CGRect(origin: view.frame.origin, size: CGSize(width: 1, height: 1))
+        alertController.popoverPresentationController?.permittedArrowDirections = .any
+
+        origin.present(alertController, animated: true, completion: nil)
+    }
+
+    private func freePhoto() -> UIAlertAction {
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift]: Maybe this func could be called `freePhotoAction` or `freePhotoAlertAction`? 
At first glance it was a bit confusing, but the return value makes it clear so it's up to you.

File: WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        })
+    }
+
+    private func files(origin: UIViewController & UIDocumentPickerDelegate) -> UIAlertAction {
+        return UIAlertAction(title: .files, style: .default, handler: { [weak self] action in
+            self?.showDocumentPicker(origin: origin)
+        })
+    }
+
+    private func cancel() -> UIAlertAction {
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift]: similar to `func freePhoto()`

File: WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift (hunk 1 above)
Comment (ctarda) [reply WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift]: I "optimized" for readability at the point of usage:
``
alertController.addAction(freePhoto())
``

That, to me, read better than 

``
alertController.addAction(freePhotoAction())
``

But I see your point. Will rename!

**Comment by ctarda** [comment]
I just pushed an update that:
- Updates the strings:
img_2626
- Fixes the crash.

The crash was tricky. It was happening because the "picker" is presented as a keyboard accessory view. When presenting other popovers, `AztecPostViewController` removes the first responder and sets it back on `viewDidDisappear` and `viewDidAppear` which are callbacks that (TIL) do not get called when presenting an alert controller.

Long story short, I have declared a delegate, which I am not sure will be completely necessary after all, but that we might want to keep around in order to tell `AztecPostViewController` what is happening in the Stock Photos flow. I would like to keep that around for now, until we move forward a little bit more and re-eavluate again if we really needed (sorry, I tend to refactor myself a lot as I go 😊 )
